mico-core/src/main/java/io/github/ust/mico/core/exception/KafkaFaasConnectorNotAllowedHereException.java

```diff
@@ -0,0 +1,10 @@
+package io.github.ust.mico.core.exception;
+
+public class KafkaFaasConnectorNotAllowedHereException extends Exception {
+
+    private static final long serialVersionUID = -6352481291495405718L;
+
+    public KafkaFaasConnectorNotAllowedHereException() {
+        super("KafkaFaasConnector is not allowed here. Use special API endpoints instead.");
```

Comment:
Here is not really specific. If possible be more specific. (also with the special endpoints)

Reply:
Ideas for a better naming?